data_structures/trie/trie.py

```diff
@@ -56,20 +84,27 @@ def print_words(node: TrieNode, word: str):  # noqa: E999 This syntax is Python
     :return: None
     """
     if node.is_leaf:
-        print(word, end=' ')
+        print(word, end=" ")
 
     for key, value in node.nodes.items():
         print_words(value, word + key)
 
 
 def test():
```

Comment:
Unfortunately, our Travis CI automated tests are [not finding the __test()__ function](https://travis-ci.org/TheAlgorithms/Python/builds/584535976#L621-L622).  Can you try renaming this function to __test_trie()__ in an attempt to satisfy the Pytest discovery https://docs.pytest.org/en/latest/goodpractices.html#test-discovery

Reply:
@cclauss Made some changes for Travis CI to detect the tests. Kindly review them. Derived from https://github.com/TheAlgorithms/Python/blob/6fa3c0b1701bc4b4a594e6b2ada335acefc12076/data_structures/binary_tree/red_black_tree.py#L675